test/yargs.js

```diff
@@ -219,7 +230,12 @@ describe('yargs dsl tests', function () {
   describe('command', function () {
     it('allows a function to be associated with a command', function (done) {
       yargs(['blerg'])
-        .command('blerg', 'handle blerg things', function () {
+        .command('blerg', 'handle blerg things', function (yargs, argv) {
+          // a fresh yargs instance for perorming command-specific parsing,
+          // and an argv instance containing the parsing performed thus far
+          // should be passed to the command handler.
+          (typeof yargs.option).should.equal('function')
```

Comment:
Should probably assert that `yargs` has been reset somehow. Maybe something like:

```js
yargs.getCommandHandlers().should.deep.equal({}) // 'blerg' command should not exist after reset
```